src/zopfli/hash.c

```diff
@@ -131,5 +131,5 @@ void ZopfliWarmupHash(const unsigned char* array, size_t pos, size_t end,
                 ZopfliHash* h) {
   (void)end;
```

Comment:
The (void)end can now be removed since the variable is now used below.

Also a question, on which input does the invalid read occur? Thank you :)